src/services/DataService.js

```diff
@@ -0,0 +1,17 @@
+export const DataService = {
+  postData: async (url = '', data = {}) => {
+    const response = await fetch(url, {
+      method: 'POST',
+      mode: 'cors',
```

Comment:
wow all of this headers are necessary? I have no idea what they are

Reply:
I removed some of them